src/trusted/service_runtime/nacl_syscall_strace.c

```diff
@@ -96,143 +96,108 @@ char * formatStringArgument(const char * input) {
   return output;
 }
 
-void NaClStraceGetpid(int cageid, int pid, long long elapsedTime) {
+void stracec_increment(int syscallnum, long long totaltime, int retval) {
+    syscallStats[syscallnum].count++;
+    syscallStats[syscallnum].totalTime += totaltime;
+    if (retval < 0) {
+        syscallStats[syscallnum].errorCount++;
+    }
+}
+
+void NaClStraceGetpid(int cageid, int pid, long long totaltime) {
   if (strace_C) {
-    syscallStats[NACL_sys_getpid].count++;
-    syscallStats[NACL_sys_getpid].totalTime += elapsedTime;
+    stracec_increment(NACL_sys_getpid, totaltime, 0);
     }else {
```

Comment:
some of these } else statements need a space between the bracket

Reply:
I have updated the spacing